addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_animation_sampler_keyframes.py

```diff
@@ -62,12 +62,18 @@ def get_target_len(self):
         return length
 
     def __set_indexed(self, value):
+        if value is None:
```

Comment:
What is the case where value come with None data?

Reply:
Perhaps it's desirable to reset `value` to its original state which is **None**.  I tried assigning None at one point to simply observe the keyframe results, but the plugin crashed. That shouldn't be an acceptable outcome.